examples/formats/NEXRAD_Level_3_File.py

```diff
@@ -8,19 +8,25 @@
 
 Use MetPy to read information from a NEXRAD Level 3 (NIDS product) file and plot
 """
+import cartopy.crs as ccrs
+import matplotlib.gridspec as gridspec
 import matplotlib.pyplot as plt
 import numpy as np
 
+from metpy.calc import azimuth_range_to_lat_lon
 from metpy.cbook import get_test_data
 from metpy.io import Level3File
-from metpy.plots import add_metpy_logo, add_timestamp, colortables
+from metpy.plots import add_metpy_logo, add_timestamp, colortables, USCOUNTIES
+from metpy.units import units
 
 ###########################################
-fig, axes = plt.subplots(1, 2, figsize=(15, 8))
+ax_rect = gridspec.GridSpec(1, 2)
+fig = plt.figure(figsize=(15, 8))
 add_metpy_logo(fig, 190, 85, size='large')
 ctables = (('NWSStormClearReflectivity', -20, 0.5),  # dBZ
            ('NWS8bitVel', -100, 1.0))  # m/s
-for v, ctable, ax in zip(('N0Q', 'N0U'), ctables, axes):
+
+for v, ctable, ax in zip(('N0Q', 'N0U'), ctables, ax_rect):
```

Comment:
So not quite fixed, yet. Here the variable `ax` is not an `Axes` object. That doesn't happen until below where you do:
```python
ax = fig.add_subplot(ax, projection=crs)
```

So here I'd make the line:

```suggestion
for v, ctable, ax_rect in zip(('N0Q', 'N0U'), ctables, spec):
```

and change above to:
```python
spec = gridspec.GridSpec(1, 2)
```